src/controllers/ControllerComplaint.ts

```diff
@@ -76,6 +77,26 @@ export default class ControllerComplaint {
 		}
 	}
 
+	async delete(req: Request, res: Response): Promise<Response> {
```

Comment:
mudem o nome dessa controller para algo mais específico para o caso dela, pfvr

Reply:
Obrigado pela revisão, nome alterado.

Reply:
Boa. E a questão das funções de delete complaint que estão aqui também? Elas são iguais as funções da outra issue de vocês? Se for, é melhor tirar daqui.

Reply:
Boa tarde, são funções com objetivos distindos que tinham o mesmo nome, agradeço pelo feedback, o nome foi alterado

Reply:
Legal. Obrigado pelo esclarecimento. Vou revisar então